hail_scripts/v01/utils/computed_fields.py

```diff
@@ -315,7 +327,7 @@ def get_expr_for_vep_consequence_terms_set(vep_transcript_consequences_root="va.
 def get_expr_for_vep_protein_domains_set(vep_transcript_consequences_root="va.vep.transcript_consequences"):
     return """%(vep_transcript_consequences_root)s
         .filter( x => isDefined(x.domains) && x.domains.length > 0 )
-        .map( x => x.domains.map( domain => domain.db+":"+domain.name ) )
+        .map( x => x.domains )
```

Comment:
because this is being called on `sortedTranscriptConsequences` instead of the root consequences, and this transformation has already been done on that field